.github/workflows/issue-pending-response.yml

```diff
@@ -0,0 +1,15 @@
+name: issue-pending-response
+on:
+  issue_comment:
+    types: [created]
+permissions:
+  issues: write
+jobs:
+  issue_commented:
+    if: ${{ !github.event.issue.pull_request  && contains(github.event.issue.labels.*.name, 'pending-response') }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: siegerts/pending-author-response@v1
```

Comment:
this must use SHA ref. See other 3rd party action usage in the repo for reference.

Reply:
3rd party action requires a AppSec consultation

Reply:
done